concordium-std/src/test_infrastructure.rs

```diff
@@ -8,7 +8,7 @@
 //!
 //! # Example
 //!
-//! ```rust
+//! ```ignore
```

Comment:
Please do not ignore. This will make `cargo doc` worse to read.

The correct way to fix this is to make it compile, but there is a separate issue for that #77 and should be done in a separate PR.

So please revert all of these.